subjects/java/checkpoints/age-finder/README.md

```diff
@@ -0,0 +1,56 @@
+## Age Finder
+
+### Instructions
+
+Create a class `AgeFinder` that provides a method to calculate the age from a given date. The date will be provided in the format `yyyy-MM-dd`.
+
+In case of any error the method `calculateAge` should return `-1`
```

Comment:
I would either specify or provide an example to show that dates in the future have to be considered errors

Reply:
I added a hint to that.